### PR TITLE
Create Blip “pydantic”

### PR DESCRIPTION
**Pull request by arnaud-fossop**
Automatically generated by Decap CMS

### CODE REVIEW

**Comment by annaelleduff** [comment]
We used it at Diana, Brock is the person to talk to!

**Comment by arnaud-fossop** [comment]
@julienblin and @annaelleduff Do you want me to add Diana then?

File: src/content/blips/pydantic.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Pydantic
+description: Pydantic is a data validation library for Python. It provides a
+  unified framework to define and validate input classes that can be used as
+  data contracts for API or CLI in pure Python.
+opinion: >-
+  Pydantic is widely used in most of the common tools in Python community
```
Comment (lvancraen) [inline src/content/blips/pydantic.md:7]:
```suggestion
  Pydantic is widely used in most of the common tools in the Python community
```

File: src/content/blips/pydantic.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  (FastAPI, Django, LangChain, etc.). It uses hints and decorators which make it
+  readable and easy to learn. It is also compatible with several schema
+  standards in REST API ecosystem (OpenAPI, JSON schema).
```
Comment (lvancraen) [inline src/content/blips/pydantic.md:10]:
```suggestion
  standards in the REST API ecosystem (OpenAPI, JSON schema).
```

**Review by lvancraen — APPROVED** [review]

**Pull request merged**